src/app/utils/CachedAxios.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+function CachedAxios() {
+  this.cache = {};
+  this.call = (url) => {
+    if (this.cache[url]) {
+      return new Promise((resolve) => {
+        resolve(this.cache[url]);
+      });
+    }
+
+    return axios(url)
+      .then((res) => {
+        this.cache[url] = res;
```

Comment:
We're letting the cache grow without bounds here, right? Which is probably fine because we're only calling `CachedAxios` from `src/app/components/SubjectHeading/BibsList.jsx`, which is only rendered client-side (meaning the cache will continue to grow until the patron refreshes the page at some point)? Want to confirm this is client-side only. This could create a problem if we forget and use it server-side.